types/module/module.go

```diff
@@ -395,22 +395,23 @@ func (m *Manager) ExportGenesisForModules(ctx sdk.Context, cdc codec.JSONCodec,
 		panic(err)
 	}
 
-	channels := make([]chan json.RawMessage, len(modulesToExport))
+	channels := make(map[string]chan json.RawMessage)
 	modulesWithGenesis := make([]string, 0, len(modulesToExport))
```

Comment:
Not for self, this is not necessary because we can iterate over the channel map.

Reply:
updated